pkg/orchestrator/event_emitter.go

```diff
@@ -33,38 +32,38 @@ func NewEventEmitter(params EventEmitterParams) EventEmitter {
 
 func (e EventEmitter) EmitJobCreated(
 	ctx context.Context, job models.Job) {
-	event := model.JobEvent{
```

Comment:
can you please add a TODO to deprecate `models.JobEvent` and this event emitter? I don't think it is used anywhere except for logging, and it should be replaced by a jobstore watcher for history events

Reply:
I'd love to remove these now, if we don't think they are being used for anything can I remove them now?

Reply:
My preference is to do it on a separate PR. This will allow you to merge this PR today, which Udit is blocked on for the python sdk work, and will simplify the review of removing the `JobEvent` models

Reply:
Sounds good.